las/client.py

```diff
@@ -172,16 +171,19 @@ def _make_request(
         signing_path: str,
```

Comment:
While we are making breaking changes I think we should rename this to just `path`. The name `signing_path` is a relic from the past when we signed the requests.
```suggestion
        path: str,
```